### PR TITLE
Upgrade groovy library for Validation Workflow

### PR DESCRIPTION
**Pull request by Divyaasm**
### Description
Pass each argument from jenkins separately to Validation groovy library 

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by rishabh6788** [comment]
This will also need a version bump, right?

**Comment by Divyaasm** [comment]
> This will also need a version bump, right?

Yes

**Comment by rishabh6788** [comment]
> > This will also need a version bump, right?
> 
> Yes

You need to update the build.gradle file for that, right?

**Comment by Divyaasm** [comment]
> > > This will also need a version bump, right?
> > 
> > 
> > Yes
> 
> You need to update the build.gradle file for that, right?

Waiting for this PR to get merged,

> > > This will also need a version bump, right?
> > 
> > 
> > Yes
> 
> You need to update the build.gradle file for that, right?

Yes Rishabh

**Review by peterzhuamazon — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
Please bump version.

File: vars/validateArtifacts.groovy [hunk 1]
```diff
@@ -9,7 +9,21 @@
 
 /**
 Wrapper that runs validation.sh script with provided args.
-@param Map[<any>] <Required> - Any arguments that you want to be passed to validation.sh script. eg: version: 1.0.0 will be passed as --version 1.0.0
+[
+@param Map[version] <Required> - Version of the opensearch artifact that needs to be validated.
+@param Map[file_path] <Required> - Url paths to Opensearch or OpenSearch DashBoards artifacts.
+Note: These parameters are mutually exclusive. Provide either of 'version' or  'file_path' parameters, to proceed with the validation.
+]
```
Comment (peterzhuamazon) [inline vars/validateArtifacts.groovy:16]: I dont think the comment here take effect?
Or is it just the github gui?

Comment (Divyaasm) [reply vars/validateArtifacts.groovy:16]: No it's due to GUI, All the info is inside the comments. We should be good.

**Pull request merged**